test/specs/login/Login.spec.js

```diff
@@ -0,0 +1,25 @@
+// login.spec.js
+var expect = require('chai').expect;
+var LoginPage = require('../../pageobjects/login.page');
+var HomePage = require('../../pageobjects/home.page');
+
+describe('login form', function () {
+    it('should deny access with wrong creds', function () {
+        LoginPage.open();
+        LoginPage.username.setValue('foo');
+        LoginPage.password.setValue('bar');
+        LoginPage.submit();
+        LoginPage.error.waitForVisible(10000); 

```

Comment:
It is not a good practice to add explicit wait in the specs 
Please move it to the page objects